### DISCUSSION TITLE
Add "sizes" attribute inside the <Img> without adding it to the query?

### DISCUSSION THREAD

**Discussion by jerehut** [gatsbyjs/gatsby#27024] [Help] [answered]
_Edit: this is specific to **Gatsby Image** plugin, didn't realize it went directly under Gatsby repo_

This might be really simple, but I can't figure out how to add the `sizes` attribute inside a fluid `<Img>` that Gatsby Image provides, none of the documentation are helping.

I'm looking to do something like this: `<Img fluid={...} sizes="(min-width: 710px) 400px, 100vw" />`
Adding the `sizes` like that just breaks the plugin, making no image show up.

How does one add the `sizes` attribute without using the query?

**Comment by pindjur** [comment, marked as answer]
gatsby image without query is useless, unless you want to add data manually.
you can pass object to fluid prop with sizes key

fluis object types

`  aspectRatio: PropTypes.number.isRequired,
  src: PropTypes.string.isRequired,
  srcSet: PropTypes.string.isRequired,
  sizes: PropTypes.string.isRequired,
  base64: PropTypes.string,
  tracedSVG: PropTypes.string,
  srcWebp: PropTypes.string,
  srcSetWebp: PropTypes.string,
  media: PropTypes.string,
  maxWidth: PropTypes.number,
  maxHeight: PropTypes.number`

**Comment by sidharthachatterjee** [comment]
@jerehut Like @pindjur correctly mentioned, sizes can't be passed in as a prop. It is meant to be part of fluid or fixed key included in the query response.
